### DISCUSSION TITLE
Need to help in using of react router tab

### DISCUSSION THREAD

**Discussion by monster1017** [community/community#41639] [Programming Help] [answered]
### Select Topic Area

Question

### Body

Hello, everyone.
I made a component called react-router-tabs using react-router-dom .
But It seems a routing don't work well.
I am looking for someone to help my issue.
Best regards.

**Comment by technexxa** [comment]
Hi, @monster1017.
I saw your article just before.
What is version of react-router-com?

**Comment by monster1017** [comment]
Hi, @Apollo-111.
Thank you for your reply.
Version is 5.0.0.

**Comment by technexxa** [comment, marked as answer]
Oh, It seems that version of react-router-dom isn't compatible with react version.

**Comment by monster1017** [comment]
Yeah, I upgraded version of react-router-com to 6.4.4.
It seems a component is going well.
@Apollo-111, thanks for your kind help.

**Comment by yonatantefera045-sketch** [comment]
Can you send me the apk
